README.md

```diff
@@ -1,2 +1,45 @@
-# data-retrieval
-Tools for downloading and processing ECMWF data for use with MSS
+# mss-data-retrieval
+Scripts to get and process ERA5 data for MSS on Linux
+
+# CDS-API Setup
+1. Create an account at https://cds.climate.copernicus.eu/user/register \
+Log into your account
+2. Navigate to https://cds.climate.copernicus.eu/api-how-to#install-the-cds-api-key \
+Copy the content of the upper black box to the right and paste it into `~/.cdsapirc`
+5. Accept https://cds.climate.copernicus.eu/cdsapp/#!/terms/licence-to-use-copernicus-products
+
+# Scripts Setup
+1. Clone this repository and move into it
+```
+git clone https://github.com/Open-MSS/data-retrieval.git
+cd data-retrieval
+```
+2. Make the shell scripts executable
+```
+chmod +x ./bin/*.sh
+```
+3. Install all requirements
+```
+pip -r install requirements.txt
+```
+4. Make sure cdo and nco are installed\
+   e.g. for Ubuntu/Debian
+```
+sudo apt-get install cdo nco netcdf-bin
+```
+
+# Usage
+1. After completing both setups, you can use this script as follows:
```

Comment:
I am not 100% sure, but in ReST the paragraph needs to be indented. It formats reasonably well as is with github, but the example is not indented properly.